### PR TITLE
versions: Kernel version bump to 4.9.27

### LINKED ISSUE #890: Update clear_vm_kernel_version on versions.txt file
A new release from kernel is out and Clear Linux will update the CC-kernel soon.

Please update the version file when https://github.com/clearlinux-pkgs/linux-container is updated.

### PR DESCRIPTION
**Pull request by jcvenegas**
Sync kernel to last lts version.

Fixes #890

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
qa-passed

**Comment by gorozco1** [comment]
This commit is already pushed here: https://github.com/01org/cc-oci-runtime/pull/672/commits/e54194c19eedea3f37e8c3add650067a014144b8

in PR #672 

We need to chose where to apply.

**Comment by jcvenegas** [comment]
@gorozco1  If we can merge #672 today, I can close this PR, If not I would like to merge this before release 2.1.8

**Comment by gorozco1** [comment]
@jcvenegas it's fine for me :)

File: versions.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ docker_ubuntu_version=16.04
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
 clear_vm_image_version=14700
-clear_vm_kernel_version=4.9.24-60
+clear_vm_kernel_version=4.9.27-63
```
Comment (gorozco1) [inline versions.txt]: Please remove the `-6x` part we do not need it it's only the counter of builds for each packaging.

Comment (jcvenegas) [inline versions.txt]: Currently I think is a dependency from CI, as they take the kernel build by clear

Comment (jcvenegas) [inline versions.txt]: @chavafg  commments?

**Comment by devimc** [comment]
lgtm

**Pull request merged**